src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -169,6 +171,68 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
   }
 
+  // Send the request's contents to Datastore in the form of an updated ScheduledInterview object.
+  // Adds a shadow.
+  @Override
+  public void doPut(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Not confident this is being called, correct me if l'm wrong. l see when someone selects an interview `selectInterview(this)` is called, but that calls the doPost()
https://github.com/googleinterns/InterviewMe/blob/5ebc6552804860cede838e785f2170cc0e720061/src/main/webapp/search-interview.js
Or if you planned to do this in another PR that's good too

Reply:
This is called within SelectInterview (the Shadow portion).